shared/domain/src/main/java/gov/va/vro/model/event/AuditEvent.java

```diff
@@ -2,21 +2,24 @@
 
 import lombok.Builder;
 import lombok.Getter;
-import lombok.ToString;
 
 import java.time.ZonedDateTime;
 
 @Getter
 @Builder(toBuilder = true)
-@ToString
 public class AuditEvent {
 
   private String eventId;
   private String routeId;
   private Class<?> payloadType;
   private Throwable throwable;
+
```

Comment:
What is the use of throwable here? If it is only for throwable != null, it is really no need to have a Throwable.

Reply:
Save a throwable in DB?